### PR TITLE
Docs updated to the new design

### PR DESCRIPTION
**Pull request by dimitarangelkov**

### CODE REVIEW

**Review by tuler — APPROVED** [review]

**Comment by tuler** [comment]
Can you squash and change the commit message to `style(docs): style update`

**Comment by dimitarangelkov** [comment]
@tuler done.

**Review by tuler — APPROVED** [review]

**Comment by dimitarangelkov** [comment]
@tuler shall I proceed with the rebase?

File: apps/docs/index.md [hunk 1]
```diff
@@ -3,24 +3,52 @@
...
+    details: The Sunodo infrastructure is a powerful tool that not only enables new use cases but also enhances existing ones by facilitating their expansion.
+    button:
+        label: Get Started
+        link: https://github.com/sunodo/sunodo
+footer:
+    logo: /logo.svg
+    navSite:
+        - link: https://sunodo.io/how-it-works
+          text: How it works?
+        - link: https://sunodo.io/sunodo-dapp-launchpad
```
Comment (tuler) [inline apps/docs/index.md:39]: We changed this to deploy right?

File: apps/docs/index.md [hunk 2]
```diff
@@ -3,24 +3,52 @@
...
+          text: Sunodo DApp Launchpad
```
Comment (tuler) [inline apps/docs/index.md:40]: Let's not use Launchpad

**Review by tuler — APPROVED** [review]
Let's merge together with the website tomorrow

**Pull request merged**